src/leaflet.filelayer.js

```diff
@@ -209,6 +210,48 @@
             }
             return layer;
         },
+		
+		/* Convert .poly into geojson and load it
+		 * The Osmosis polygon filter file format is supported by Osmosis, mapsplit, 
+		 * osmconvert, osmchange and pbftoosm as a way of defining extraction polygons.
+		 * https://wiki.openstreetmap.org/wiki/Osmosis/Polygon_Filter_File_Format
+		 */
+        _loadPoly: function _loadPoly(content) {
+			var lines = content.split("\n");
+			var json = {
+				"type": "Feature",
+				"name": $.trim(lines[0]),
```

Comment:
There's no need of JQuery for this, `line[0].trim()` should work https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim

Reply:
No more jquery